src/main/java/frc/robot/subsystems/lifter/factory/LifterRealConstants.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.lifter.factory;
+
+import com.ctre.phoenix6.SignalLogger;
+import com.ctre.phoenix6.configs.FeedbackConfigs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.robot.constants.GlobalConstants;
+import frc.robot.constants.IDs;
+import frc.robot.hardware.motor.phoenix6.TalonFXMotor;
+import frc.robot.hardware.motor.phoenix6.TalonFXWrapper;
+import frc.robot.hardware.signal.phoenix.Phoenix6SignalBuilder;
+import frc.robot.subsystems.lifter.LifterStuff;
+import frc.utils.AngleUnit;
+
+import static edu.wpi.first.math.util.Units.inchesToMeters;
+import static edu.wpi.first.units.Units.Seconds;
+import static edu.wpi.first.units.Units.Volts;
+
+public class LifterRealConstants {
+
+	private static final double DRUM_RADIUS = inchesToMeters(0.96);
+
+	private static TalonFXConfiguration getConfiguration() {
+		TalonFXConfiguration talonFXConfiguration = new TalonFXConfiguration();
+
```

Comment:
add current limits and maybe more

Reply:
i dont want to add current limits because you need this motor to pull a lot of load, thus drawing a lot of current